src/main/java/co/rsk/federate/signing/hsm/message/BlockHeaderBrother.java

```diff
@@ -0,0 +1,29 @@
+package co.rsk.federate.signing.hsm.message;
+
+import org.ethereum.core.BlockHeader;
+import org.spongycastle.util.encoders.Hex;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class BlockHeaderBrother {
```

Comment:
A block header brother is not a list of brothers. A block header brother is another block header.

What I was thinking is a class that represents and connects the block headers with their list of brothers to send to the HSM.

Something like
```
class ParsedHeader {
String encoded;
List<String> encodedUncles;
```

And the parsing could be done in this class also.

That way AdvanceBlockchainMessage class only has a list parsed blocks and the headers are connected to their uncles.

What do you think? Any other ideas? Any other naming ideas?